interop/xds/client/client.go

```diff
@@ -163,6 +213,49 @@ func (s *statsService) GetClientStats(ctx context.Context, in *testpb.LoadBalanc
 	}
 }
 
+func (s *statsService) GetClientAccumulatedStats(ctx context.Context, in *testpb.LoadBalancerAccumulatedStatsRequest) (*testpb.LoadBalancerAccumulatedStatsResponse, error) {
+	return accStats.buildResp(), nil
+}
+
+type configureService struct {
+	testpb.UnimplementedXdsUpdateClientConfigureServiceServer
+}
+
+func (s *configureService) Configure(ctx context.Context, in *testpb.ClientConfigureRequest) (*testpb.ClientConfigureResponse, error) {
+	types := make(map[string]bool)
+	for _, typ := range in.GetTypes() {
+		types[typ.String()] = true
+	}
+	rpcsToMD := make(map[string][]string)
+	for _, md := range in.GetMetadata() {
```

Comment:
Not all RPCs have metadata to send.
If an RPC doesn't have metadata, will it end up in `rpcsToMD`?

Might as well move this conversion into a function, and cover it in a unit test.

Reply:
Made this change. Now the metadata will be the result of `metadata.Pairs(md...)` with `md` being an empty slice.